webpack/containers/DashboardContainer.js

```diff
@@ -154,23 +155,39 @@ class DashboardContainer extends Component {
       searchTerms = null;
     }
     this.setState({searchTerms: searchTerms, progFilters: progFilters, sysFilters: sysFilters});
-    this.props.fetchSearchResults(searchTerms, searchType, progFilters, sysFilters);
+    this.props.fetchSearchResults(searchTerms, searchType, progFilters, sysFilters, this.state.myStuffFilter);
   }
 
-  selectType(searchType) {
+  selectType(searchType, myStuffToggle=false) {
     let searchTerms = null;
+    let myStuffFilter = false;
     if(this.state.searchTerms === '') {
```

Comment:
Nitpicking style, I think this is better:
 if(this.state.searchTerms !== ''){
searchTerms = this.state.searchTerms;
}